12/umbraco-cms/reference/templating/macros/managing-macros.md

```diff
@@ -74,7 +74,7 @@ This list defines the different types of macro parameters:
 * Checkbox(`Umbraco.TrueFalse`) - A true/false value
 * Content Type picker (`contentType`) - Choose a type from existing Document Types
 * Decimal (`Umbraco.Decimal`) - accepts only numbers
-* Email address (`Umbraco.EmailAddress`) - validates to valid email adresses
+* Email address (`Umbraco.EmailAddress`) - validates to valid email addresses
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?